codehawk.properties

```diff
@@ -1,20 +1,20 @@
 category=External Analysers
 description=Analyze Java code smell
 homepageUrl=https://github.com/SDPMLab/CodeHawk
-archivedVersions=1.4
-publicVersions=1.5
+archivedVersions=1.5
+publicVersions=1.6
 
 defaults.mavenGroupId= org.codehawk.plugin
 defaults.mavenArtifactId= codehawk
 
 1.5.description= Added new rules
 1.5.sqVersions=[7.9,LATEST]
```

Comment:
Assuming this _is_ for 1.6, this range needs to start with 8.9. Nothing below that is supported.